src/components/executive-education-2u-error/ExecutiveEducation2UErrorPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ArrowBack } from '@edx/paragon/icons';
+import { Button, Col } from '@edx/paragon';
+import {
+  ErrorPage,
+} from '../error-page';
+
+const ExecutiveEducation2UErrorPage = ({ failureReason, httpReferrer }) => {
```

Comment:
nit: perhaps `ExecutiveEducation2UError` since this component isn't rendering an entire page (including header/footer/etc.), but just a subset of the page.